_rules/descriptive-heading-b49b2e.md

```diff
@@ -155,29 +165,13 @@ Heading marked up with `role="heading"` that does not describe the topic or purp
 
 #### Failed Example 4
 
-Heading marked up with `h` element that does not describe the topic or purpose of its section of the content. The heading is visible, but is not included in the Accessibility Tree.
+Heading marked up with `h1` element that does not describe the topic or purpose of its section of the content. The heading is visible, but is not included in the accessibility tree.
 
 ```html
 <h1 class="target" aria-hidden="true">Weather</h1>
 <p>We are open Monday through Friday from 10 to 16</p>
 ```
 
-#### Failed Example 5
```

Comment:
I moved these. Because these headings have no content, they aren't "visible", and so they're not applicable. I think this is correct, since headings without content can be ignored by assistive technologies.